pre_install_report/test/test_data/json_data/azure_nodes_no_master.json

```diff
@@ -1639,7 +1639,7 @@
                     },
                     {
                         "names": [
-                            "gcr.io/google-samples/node-hellofhds058d238d0ab54efb76ec0f7b1da666cefa9b40be59ef34346a761b8adc2dd45459b"
+                            "cr.io/my-samples/node-hellofhds058d238d0ab54efb76ec0f7b1da666cefa9b40be59ef34346a761b8adc2dd45459b"
```

Comment:
Why do we retain any reference to a node-hello image?   Dup this comment a few times more in the changeset.

Reply:
The node-hello reference in pytest data files was unrelated to our removal of the functionality that actually uses that image.   That image was chosen likely as a conveniently known image name to stick in test json file.   Different references now.